api/src/main/kotlin/io/spine/protodata/renderer/SourceFile.kt

```diff
@@ -269,6 +269,9 @@ private constructor(
         return text(code)
     }
 
+    /**
+     * Adds an action to be executed before obtaining the [text] of this source file.
+     */
     internal fun whenRead(action: (SourceFile) -> Unit) {
```

Comment:
I understand "when read", as "when it has been read". So, after the operation is done. 

Can we rename it to `beforeRead`?